.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
```

Comment:
the `sys.dont_write_bytecode = True` is supposed to keep these out.  did that not work?

Reply:
Hm. I moved that line into `main()` because it looked silly in the imports and I didn't know what it did. But apparently it needs to be before the import of `validate_deprecated_mapping` to stop it generating `validate_deprecated_mapping.pyc`. Is there any particular reason to suppress generating `.pyc` files other than cluttering up the directory? Because I would assume there are (minor) performance benefits from keeping them.

Reply:
ya good point, better to just delete that line.